templates/build/news-feed-base.build.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="new-news-feed-list-container loading">
+<div class="new-news-feed-list-container ready">
```

Comment:
You removed the initial loading state? Why?

Reply:
That was a test, gonna get it back and just remove the loading state during the first iteration only. Nice catch!